htdocs/knowledgemanagement/knowledgerecord_card.php

```diff
@@ -332,6 +353,15 @@
 	//unset($object->fields['fk_soc']);					// Hide field already shown in banner
 	include DOL_DOCUMENT_ROOT.'/core/tpl/commonfields_view.tpl.php';
 
+	//Language of question/response
```

Comment:
Can you include this into the commonfields_view.tpl.php so every module that use a 'lang' field will benefit the language selector/beautifier.